### ISSUE TITLE
Integration with GWT

### ISSUE THREAD

**Issue by NicolasBerino** [artf/grapesjs#1073]
Hello, 
Actually, i've a gwt application and i need to integrate grapes js. My problem is i not really how to do this because of all the different files in JS. I understood that it was necessary to create a module with specific path but i don't really see how to link js with gwt in the context of such a large project. If somenone could give me a hand, it would be nice.

**Comment by artf** [comment]
I don't know how GWT works but I'd not expect anything different from using any other external library. You just need to include one grapesjs js file https://unpkg.com/grapesjs@0.14.9/dist/grapes.min.js (probably you will need a [style](https://unpkg.com/grapesjs@0.14.9/dist/css/grapes.min.css) too) and then use it inside the correct context.

**Comment by NicolasBerino** [comment]
thank you for your answer, but it does not help me because it is at the level of writing the context that I have trouble. In fact, I tried to use the readme to integrate the two min files (with minimal context as a result) but it did not work.
If someone had tried to look for an integration of grapesJS in gwt and could give me some guidance on how to do this would be nice. I tried with JSInterop but I struggle to see how it works.
